server/service/src/sync/test/merge_helpers.rs

```diff
@@ -0,0 +1,26 @@
+use repository::{
+    mock::{mock_name_link_from_name, MockDataCollection},
+    NameLinkRowRepository, NameRow, RepositoryError, StorageConnection,
+};
+
+pub fn name_links_merged(
+    connection: &StorageConnection,
+    mock_data: &MockDataCollection,
+) -> Result<(), RepositoryError> {
+    let name_link_repo = NameLinkRowRepository::new(&connection);
+
+    let names: Vec<NameRow> = mock_data
+        .data
+        .iter()
+        .map(|(_, mock)| mock.names.clone())
+        .flatten()
+        .collect();
+
+    for name in names {
+        let mut name_link = mock_name_link_from_name(&name);
```

Comment:
Ups, I think I deleted `mock_name_link_from_name` after some refactoring/cleaning because it wasn't used anymore...

Reply:
Updated from base to get the compiler upset, v true. Request changes!

Reply:
At the moment `mock_name_link_from_name` is only used once could you please inline it? I don't think we need a helper method for it...